<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formImportData</class>
 <widget class="QDialog" name="formImportData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Surface</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Importar datos desde:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radio1">
        <property name="text">
         <string>Google Elevation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio2">
        <property name="text">
         <string>Archivo DEM</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="gbLocalFile" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="editDEM"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonDEM">
        <property name="text">
         <string>Sel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
